src/public/assets/data/partners.json

```diff
@@ -2,110 +2,112 @@
   "partners": [{
     "name": "Australian Government",
     "logo": "australian-government-department-of-social-services.jpg",
-    "type": "program partner",
     "intro": "At the beginning of 2016, the Australian Government Department of Social Services partnered with Red Dust to provide support for the Strong Young Women's Program.",
     "url": "https://www.dss.gov.au/"
   }, {
+    "name": "Nike",
+    "logo": "nike.jpg",
+    "intro": "Nike Australia has established a long and significant relationship with Red Dust Role Models.As a founding corporate partner, Nike Australia has seen Red Dust Role Models develop into an effective program that sees the lives of indigenous youth enriched.Nike believes in the power of sport to unleash potential.",
+    "url": "http://www.nike.com/"
+  }, {
+    "name": "Network Ten",
+    "logo": "ten.jpg",
+    "intro": "Ten Network Holdings is committed to supporting and giving back to the communities in which we operate and helping our staff find ways to work with a diverse range of foundations and not-for-profit organisations including Red Dust Role Models.",
+    "url": "https://tenplay.com.au/"
+  }, {
+    "name": "Coles Express",
+    "logo": "coles-express.jpg",
+    "intro": "Coles Express has partnered with Red Dust Role Models since 2014. Working in remote Indigenous communities​ across the Top End for more than 20 years, Red Dust Role Models 'Healthy Living' programs encourage Indigenous youth to learn more about health, and inspire them to live healthier lifestyles.",
+    "url": "https://www.colesexpress.com.au/"
+  }, {
     "name": "Danone",
     "logo": "danone-nutricia.png",
-    "type": "program partner",
```

Comment:
Request from client says that they don't want that specific included anymore because it makes things too political, and clients will often change from being 'barely involved' to having recently donated and requiring 'honoring' so it is easier to remove them